src/plone/restapi/profiles/testing/types.xml

```diff
@@ -6,5 +6,6 @@
  <object name="ATTestFolder"
     meta_type="Factory-based Type Information with dynamic views"/>
  <object name="DXTestDocument" meta_type="Dexterity FTI" />
+ <object name="DXTestFolder" meta_type="Dexterity FTI" />
```

Comment:
@thet would it be possible to get away without adding that text folder here? I personally really dislike testing profiles because they hide the test setup in a remote place instead of making it obvious in the test itself what is set up. They are also super inflexible. If you change something in the testing profile you always risk to break multiple tests. Getting rid of the test profile has been on my todo list for quite some time and I would like to avoid adding additional work here for me. This is not your fault, of course, since the testing profile was there already. You would do me a personal favor if you could refactor that. :)

Reply:
I can easily get rid of the DXTestFolder when changung DXTestDocument's base class from ``Item`` to ``Container``.
I can also check if I can just use Folder+DXTestDocument. Should be possible if I can leave the ``title`` without a value for testing.